File: friend-service/src/main/java/vn/iostar/friendservice/dto/response/FriendResponse.java
package vn.iostar.friendservice.dto.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class FriendResponse {
    private String userId;
    private String background;
    private String avatar;
    private String username;
    private Boolean isOnline;
}

File: conversation-service/src/utils/DTO/FriendResponse.js
class FriendResponse {
  constructor(friend) {
    this.userId = friend.userId;
    this.background = friend.background || null;
    this.avatar = friend.avatar || null;
    this.username = friend.username || null;
    this.isOnline = friend.isOnline || null;
  }
}

module.exports = FriendResponse;
